src/vue-components/lookup.vue

```diff
@@ -113,7 +126,13 @@
         if (this.lookuptext.length === 0) {
           return null
         }
-        let languageID = LanguageModelFactory.getLanguageIdFromCode(this.options.items.lookupLanguage.currentValue)
+        let languageID
+
+        if (!this.overrideLanguage) {
+          languageID = LanguageModelFactory.getLanguageIdFromCode(this.options.items.lookupLanguage.currentValue)
```

Comment:
I think this should be 
`LanguageModelFactory.getLanguageIdFromCode(this.options.items.preferredLanguage.currentValue`

If the overrideLanguage checkbox is not checked we want to use to the default preferred page language.

Reply:
If we use `LanguageModelFactory.getLanguageIdFromCode(this.options.items.preferredLanguage.currentValue` for overrideLanguage - than the user won't be able to change language through select boxes simply clicking **Using language**.

May be I haven't understood correctly the workflow?
Do we need **Using language** link with overrideLanguage chackbox?
We won't use parent language in lookup as before?

Reply:
oh, no, I think maybe this is my mistake then.  Please go ahead and merge as it is, and if Monica wants something different she will tell us! Thank you!